src/tests/ftest/recovery/ms_membership.py

```diff
@@ -79,74 +79,54 @@ def test_checker_on_admin_excluded(self):
         dmg_command.system_start(ranks="1")
 
     def test_enable_disable_admin_excluded(self):
-        """Test admin can enable and disable the rank state to AdminExcluded when the
-        rank is down.
+        """Test dmg system exclude and clear-exclude.
+
+        Test admin can enable and disable the rank state to AdminExcluded when the rank is
+        down.
 
         1. Stop rank 1.
-        2. Set rank 1 to AdminExcluded state by calling dmg system exclude --ranks=1.
-        3. Verify that the rank 1 state is AdminExluded by calling dmg system query.
-        4. Verify that the checker can be run with AdminExcluded state by calling enable,
+        2. Set rank 1 to AdminExcluded by calling dmg system exclude --ranks=1 and verify
+        the state has been changed.
+        3. Verify that the checker can be run with AdminExcluded state by calling enable,
         start, query, and disable. Verify that none of the commands returns error.
-        5. Disable AdminExcluded by calling dmg system clear-exclude --ranks=1
-        6. Verify that the rank 1 state is Excluded.
-        7. Start rank 1 for clean up. (At this point, its state is "Excluded".)
+        4. Disable AdminExcluded of rank 1 by calling dmg system clear-exclude --ranks=1
+        and verify the state has been changed.
+        5. Servers haven't been started, so update the expected state of rank 0 for
+        cleanup.
 
         Jira ID: DAOS-11704
 
         :avocado: tags=all,weekly_regression
         :avocado: tags=vm
-        :avocado: tags=recovery,pass_0
-        :avocado: tags=test_enable_disable_admin_excluded
+        :avocado: tags=recovery,ms_membership
+        :avocado: tags=MSMembershipTest,test_enable_disable_admin_excluded
         """
         errors = []
         dmg_command = self.get_dmg_command()
 
         # 1. Stop rank 1.
         dmg_command.system_stop(ranks="1")
 
-        # 2. Set rank 1 to AdminExcluded state.
-        dmg_command.system_exclude(ranks="1", rank_hosts=None)
-
-        # 3. Verify that the rank 1 state is AdminExluded.
-        query_out = dmg_command.system_query()
-        admin_excluded_found = False
-        rank_1_state = None
-        for member in query_out["response"]["members"]:
-            if member["rank"] == 1:
-                rank_1_state = member["state"]
-                if rank_1_state == "adminexcluded":
-                    admin_excluded_found = True
-                break
-        if not admin_excluded_found:
-            self.fail("Rank 1 state is not AdminExcluded!: {}".format(rank_1_state))
+        # 2. Set rank 1 to AdminExcluded and verify the state has been changed.
+        self.server_managers[-1].system_exclude(ranks=[1], daos_log=self.log)
```

Comment:
`self.log` is not a `DaosLog` object.  Did you intend to use `self.test_log.daos_log`?

Reply:
I removed `daos_log` from `system_exclude()`. Thanks.